dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/resources/docker/basic/docker-compose.yaml

```diff
@@ -32,6 +32,27 @@ services:
       interval: 5s
       timeout: 60s
       retries: 120
+#    command:
+#      - /bin/bash
+#      - -c
+#      - 'sed -i "s|resource.storage.type=.*|resource.storage.type=hdfs|g" /opt/dolphinscheduler/standalone-server/conf/common.properties'
+  hdfs:
+    image: mdouchement/hdfs:latest
+    hostname: hdfs
+    ports:
+      - "22022:22"
```

Comment:
```suggestion
      - 22022:22
```

Reply:
In docker compose, those with port numbers less than 60 need to be enclosed in quotation marks.Otherwise, it will cause an exception.

Reply:
Thanks for your clarification, I found out this in https://docs.docker.com/compose/compose-file/compose-file-v3/#ports